api/app.py

```diff
@@ -80,6 +80,28 @@ def search_courses(query):
     course_equivalencies = course_equivalencies_join_to_dict(result)
     return jsonify(course_equivalencies)
 
+@app.route('/course/search', methods=['POST', 'GET'])
+def course_search():
+    if request.method == 'POST':
+        print("HI LOL")
+        uw_course = request.form['uw_course_name']
+        program = request.form['program']
+        year = request.form['year_taken']
+        uni = request.form['host_uni']
+        course_name = request.form['host_course_name']
+        course_code = request.form['host_course_code']
+
+        uni = University.query.filter(University.name == uni).first()
+        uni_id = uni.id if uni else 54
+        uw_course = UWCourse.query.filter(UWCourse.code == uw_course).first()
+        uw_course_id = uw_course.id if uw_course else 1
+
+        db.session.add(CourseEquivalency(uwcourse_id=uw_course_id,university_id=uni_id,code="{0}: {1}".format(course_code,course_name), year_taken=year,student_program=program))
+        db.session.commit()
+        return redirect(url_for('course_search'))
+    else:
+        return jsonify("")
```

Comment:
Why does the GET return nothing? if it's not needed could we just remove it or is this a placeholder for a future task?

Reply:
yup deleting